src/ui/TotalsNumber/TotalsNumber.spec.tsx

```diff
@@ -29,6 +30,7 @@ describe('TotalsNumber', () => {
     })
 
     it('renders - when there is an invalid value', () => {
+      // @ts-expect-error
```

Comment:
Can we let value be undefined on the type in setup? Prefer avoiding heavy use of expect-error